### PR TITLE
Scenario cleanup

### PR DESCRIPTION
**Pull request by naterush**
# Changes

Upgrades the scenarios to actually use the TEST_URL, and fix a bug where the remote URL wasn't being changed to the TEST_URL upon scenario generation.

- [ ] I ran all the tests to make sure I didn't break anything.

### CODE REVIEW

**Comment by naterush** [comment]
@aarondr77 this fixes a bug in scenario creation where TEST_URL wasn't being set properly. LGTM before we do anything else. It also fixes all the scenarios that have real URLs.

Once this is merged, if you made an scenarios in the meantime, you can either:
1. Recreate your scenarios.
2. If there's a bunch, use the upgrade process that is merged in here. You just have to click the upgrade button in the dev screen, and then watch the console. It will print out any scenarios you need to upgrade.

File: src/saga/upgrade.js [hunk 1]
```diff
@@ -1,26 +1,34 @@
...
-    versionRange.values = [["0.0.1"]];
-
-    await context.sync();
+    const project = new Project(context);
+
+    const remoteURL = await project.getRemoteURL();
+
+    if (remoteURL !== TEST_URL) {
+        // if the remote URL isn't the test URL, we update it
+        await project.setRemoteURL(remoteURL);
```
Comment (naterush) [inline src/saga/upgrade.js]: This is where the upgrade happens, btw.

File: src/taskpane/components/DevScreen.js [hunk 2]
```diff
@@ -28,7 +28,7 @@ async function createScenario() {
     // First, we make sure we're using the test url, so we don't sync things to the scenario
     await Excel.run(async (context) => {
         const project = new Project(context);
-        project.setRemoteURL(TEST_URL);
+        await project.setRemoteURL(TEST_URL);
```
Comment (naterush) [inline src/taskpane/components/DevScreen.js:31]: This is what the bug was; we weren't waiting on the promise to resolve.

**Comment by aarondr77** [comment]
@naterush I pulled this and tried to update scenarios, it printed out the new scenarios, I copied them over to update the scenarios. But when I run it again, it says that my scenarios still need to be updated. 

Should I merge this in first and then upgrade like you said??

**Comment by aarondr77** [comment]
It looks like you already updated the scenarios in 8e3df39a0b77f3f3b514eceb5ce83822581c0e0f already also. But it still told me to update them.

**Comment by naterush** [comment]
@aarondr77 ok, scenarios should be updated. There was a bug in setRemoteURL, and also a bug in upgrade that compounded eachother.

**Comment by aarondr77** [comment]
It worked! Merging.

**Pull request merged**